packages/openlmis/src/Utils.js

```diff
@@ -0,0 +1,88 @@
+import nodepath from 'node:path';
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  makeBasicAuthHeader,
+} from '@openfn/language-common/util';
+
+export const authorize = state => {
```

Comment:
I've been running some tests locally and I notice the auth step is super slow - at least when failing.

Can we add a log line to say "logging in as <username>" or "logging in with access token" or something?

I'm sure you added that originally? Maybe it was a different PR because I can't see it now.

Anyway, a little bit of feedback here would be super helpful

Reply:
I wonder if we can be putting out a more useful error message as well. Can we print the client id and secret? Or warn if one is un-set?

Reply:
I have added logs for `logging`, `clientId` and `clientSecret`

Reply:
That's great, nice one